app/swagger/requests/in_progress_forms.rb

```diff
@@ -55,7 +55,10 @@ class InProgressForms
         operation :get do
           extend Swagger::Responses::AuthenticationError
 
-          key :description, 'Get form data'
+          key :summary,     'Get form data'
+          key :description, 'When a GET request is made for a user completing disability claim form 21-526EZ
+            for the first time, the API service will make a call to add the user to the Master Veteran Index (MVI)
+            and a new blank form will be served to the client.'
```

Comment:
This is close but not quite correct. The proxy add is asking MVI to make a request to birls and/or corp database and asking them create identities there. This will populate corp/birls ids in the MVI which are both required to make a successful 526 submission through EVSS. The veteran *should* already exist in MVI. Once this action has been done, the in progress controller can move forward and serve up the form and attempt to prefill the data as it normally does.

Reply:
Thanks for catching this. I wrote a new description. Could you do another pass on it for me please?

Reply:
Looks solid!